docs/language-usage.rst

```diff
@@ -153,7 +154,7 @@ metadata and other properties.
 
      >>> text_content = ("Michelangelo Caravaggio, Italian painter, is "
      ...                 "known for 'The Calling of Saint Matthew'.")
-     >>> document = client.document(text_content)
+     >>> document = language.document.Document(client, content=text_content)
```

Comment:
1. `language.Document` exists in `__init__.py` for the package
1. Use one of the `client.document_*` factory constructors?